code/datums/mood_events/generic_negative_events.dm

```diff
@@ -202,56 +140,28 @@
 /datum/mood_event/sad_empath/add_effects(mob/sadtarget)
 	description = "<span class='warning'>[sadtarget.name] seems upset...</span>"
 
-/datum/mood_event/artbad
-	description = "<span class='warning'>I've produced better art than that from my ass.</span>\n"
-	mood_change = -2
-	timeout = 2 MINUTES
-
 /datum/mood_event/sacrifice_bad
 	description ="<span class='warning'>Those darn savages!</span>"
 	mood_change = -5
 	timeout = 2 MINUTES
 
-/datum/mood_event/artbad
-	description = "<span class='warning'>I've produced better art than that from my ass.</span>"
-	mood_change = -2
-	timeout = 2 MINUTES
-
 /datum/mood_event/gates_of_mansus
 	description = "<span class='boldwarning'>LIVING IN A PERFORMANCE IS WORSE THAN DEATH</span>"
 	mood_change = -25
 	timeout = 4 MINUTES
 
-//These are unused so far but I want to remember them to use them later
-/datum/mood_event/cloned_corpse
-	description = "<span class='boldwarning'>I recently saw my own corpse...</span>"
-	mood_change = -6
-
-/datum/mood_event/surgery
-	description = "<span class='boldwarning'>HE'S CUTTING ME OPEN!!!</span>"
-	mood_change = -8
-
 /datum/mood_event/nanite_sadness
 	description = "<span class='warning robot'>+++++++HAPPINESS SUPPRESSION+++++++</span>"
 	mood_change = -7
 
 /datum/mood_event/nanite_sadness/add_effects(message)
 	description = "<span class='warning robot'>+++++++[message]+++++++</span>"
 
-/datum/mood_event/sec_insulated_gloves
```

Comment:
This looks like a pseudo-mechanical way to disincentivize sec from grabbing insuls and tools to break into places (Assuming that it actually gets applied correctly)

It's probably inconsequential enough (only -1 mood) that it doesn't matter, but definitely something to consider.